packages/khalti_flutter/lib/src/util/url_launcher_util.dart

```diff
@@ -35,6 +35,7 @@ class UrlLauncherUtil {
       await launcher.launchUrl(
         Uri.parse(url),
         webOnlyWindowName: openInNewTab ? null : '_self',
+        mode: launcher.LaunchMode.externalApplication,
```

Comment:
Shouldn't this be browser application ?

Reply:
there is no `browserApplication` launch mode.